scripts/sct_extract_metric.py

```diff
@@ -369,6 +369,8 @@ def main(fname_data, path_label, method, slices_of_interest, vertebral_levels, f
             normalizing_label[0], slices_list = remove_slices(normalizing_label[0], slices_of_interest)
         if fname_mask_weight:  # if the flag -mask-weighted was specified,
             im_weight.data, slices_list = remove_slices(im_weight.data, slices_of_interest)
+    else:
+        slices_list = range(0, nz)
```

Comment:
I suggest using `np.zeros(nz)`, because `range` is an iterable in python3 which would cause break the code.